config.json

```diff
@@ -1,9 +1,5 @@
 {
   "bootstrap": [
-    "connectora.ringr2.devsca.com",
-    "connectorb.ringr2.devsca.com",
-    "connectorc.ringr2.devsca.com",
-    "connectord.ringr2.devsca.com",
-    "connectore.ringr2.devsca.com"
+      "bucketclient.testing.local:9000"
```

Comment:
YES ! Finally rid of our connector's FQDNs from the test configs.. Also, this means we never used our dummy server, right ? :/